src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -264,38 +264,49 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
 
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const handleInputChange = useCallback(
-    debounce(async (inputValue: string) => {
-      setValue(SendFormFields.SendMax, false)
-      const key =
-        fieldName !== SendFormFields.FiatAmount
-          ? SendFormFields.FiatAmount
-          : SendFormFields.CryptoAmount
-      const amount =
-        fieldName === SendFormFields.FiatAmount
-          ? bnOrZero(bn(inputValue).div(price)).toString()
-          : bnOrZero(bn(inputValue).times(price)).toString()
-
-      setValue(key, amount)
+    debounce(
+      async (inputValue: string) => {
+        setLoading(true)
+        setValue(SendFormFields.SendMax, false)
+        const key =
+          fieldName !== SendFormFields.FiatAmount
+            ? SendFormFields.FiatAmount
+            : SendFormFields.CryptoAmount
+        const amount =
+          fieldName === SendFormFields.FiatAmount
+            ? bnOrZero(bn(inputValue).div(price)).toString()
+            : bnOrZero(bn(inputValue).times(price)).toString()
+
+        setValue(key, amount)
+
+        const estimatedFees = await estimateFormFees().catch(e => {
```

Comment:
Curious on the combined use of `await` and the `catch` handler. Shouldn't this use `try/catch` with `await`?

Reply:
awaited promises are catchable, so that allows us to avoid to first initialise the variable to then conditionally assign it on the catch block !

Reply:
But @cjthompson has a valid point that this is an inconsistent way to handle errors.
I still think there's value in this specific case as it makes things slightly more readable and avoids one branch, but that's probably nitpicking.

Changed it to a try/catch block!